src/iconsManifest/supportedExtensions.ts

```diff
@@ -718,6 +718,12 @@ export const extensions: IFileCollection = {
       ],
       format: FileFormat.svg,
     },
+    {
+      icon: 'ceylon',
+      extensions: ['ceylon'],
```

Comment:
@robertohuertasm See what I mean?

Reply:
This would happen anyway. Indeed, we haven't even merged the conflicting PR yet.

Reply:
Would not happen if we didn't start having gray zones with this [PR](https://github.com/vscode-icons/vscode-icons/pull/2097). Fortunately, we resolved that but the consequences have been shown immediately.

Reply:
I don't see it that way. I don't even see the direct connection tbh. I don't think @maheshuligade has even looked into that PR. He will have to tell us. 😉 Anyway, this is an easy fix. In this case, the extension makes total sense as it's syntax highlighting, so the extension support must be removed.

Reply:
I'm confident that he was misled by that PR.